transmonee_dashboard/src/transmonee_dashboard/index.py

```diff
@@ -2,7 +2,7 @@
 
 from .app import app
 from .utils import DashRouter, DashNavBar
-from .pages import education, child_protection, home, child_health
+from .pages import education, child_protection, home, child_poverty
```

Comment:
here we should have both child poverty and health as a result! The same applies for other changes in `index.py` below

Reply:
When I first took the branch for poverty, it was before you merged health... this is why. I guess when you merge poverty to master, health will also be there.

Reply:
yes! Would like to know how this is sorted out in git flow =) . We would need in some part of the code, a sort of appends rather than replaces ...